draft-ietf-quic-multipath.md

```diff
@@ -1028,16 +1024,16 @@ ACK_MP frame is formatted as shown in {{fig-ack-mp-format}}.
 Compared to the ACK frame specified in {{QUIC-TRANSPORT}}, the following
 field is added.
 
-Path Identifier:
-: The identifier of the path to identify the packet number
+Receiver's Destination Connection ID Sequence Number:
+: The sequence number of the Connection ID identifying the packet number
   space of the 1-RTT packets which are acknowledged by the ACK_MP frame.
 
-If an endpoint receives an ACK_MP frame with a Path
-ID which was never issued by endpoints (i.e., with a sequence number
+If an endpoint receives an ACK_MP frame with a Connection ID sequence
+number which was never issued (i.e., with a sequence number
 larger than the largest one advertised), it MUST treat this as a connection
 error of type MP_PROTOCOL_VIOLATION and close the connection.
-If an endpoint receives an ACK_MP frame with a Path ID
-which is no more active (e.g., retired by a RETIRE_CONNECTION_ID
+If an endpoint receives an ACK_MP frame with a Connection ID sequence
+number which is no more active (e.g., retired by a RETIRE_CONNECTION_ID
 frame or belonging to closed paths), it MUST ignore the ACK_MP frame
 without causing a connection error.
 
```

Comment:
The reference to "closed paths" is a bit ambiguous, because it is in fact OK to send an ACK_MP after and ABANDON_PATH -- or even in the same packet, but QUIC does not specify the order of processing of frames in a packet. And it is also OK to process it, rather than spuriously repeat data that has been actually received. I would be tempted to just strike out "or belonging to closed paths". Or maybe strike out the whole parenthesis. Or maybe strike out the whole paragraph about not active anymore. For comparison, Section 19.16 of RFC 9000 just says:

Receipt of a RETIRE_CONNECTION_ID frame containing a sequence number greater than any previously sent to the peer MUST be treated as a connection error of type PROTOCOL_VIOLATION.

Reply:
I think your point about the whole parenthesis is right. Also, this issue about seeing retired DCID also applies in PATH_ABANDON and PATH_STATUS (i.e., just ignore the frame), so we may just want to add a sentence at the beginning of the presentation of the multi path-specific frames?

Reply:
I see that this text was remove, which is fine. But was it re-added somewhere else?

Reply:
Not yet. We may add a sentence to clarify that we should just ignore Multipath frames related to retired DCIDs, but I would suggest to add such a sentence at the beginning of the section introducing these frames, as this applies to all of them.

Reply:
Okay, maybe let's open an issue, so we don't forget!

Reply:
Filled #193.